src/client/method-status.c

```diff
@@ -26,7 +26,7 @@ static const struct bus_properties_map property_map[] = {
         { "FreezerState", offsetof(unit_info_t, freezer_state) },
         { "SubState", offsetof(unit_info_t, sub_state) },
         { "UnitFileState", offsetof(unit_info_t, unit_file_state) },
-        {}
+        { NULL, 0 },
```

Comment:
So, that's annoying... I wouldn't format it if you don't have an expression of the `offset(x,y)` format in the second row.  Not really relevant to this PR